authentication/urls.py

```diff
@@ -1,13 +1,16 @@
 from django.urls import path
-from authentication.views import RegisterView, LoginView, SendVerificationEmailView, SendRecoverPasswordEmailView
+from authentication.views import RegisterView, LoginView, SendVerificationEmailView, SendRecoverPasswordEmailView, UpdateProfileView
+from rest_framework_simplejwt.views import TokenObtainPairView
 from rest_framework_simplejwt.views import TokenRefreshView
 
 app_name = 'authentication'
 
 urlpatterns = [
     path('register/', RegisterView.as_view(), name='register'),
     path('login/', LoginView.as_view(), name='login'),
+    path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
     path('verify/', SendVerificationEmailView.as_view(), name='verify_email'),
     path('recover/', SendRecoverPasswordEmailView.as_view(), name='recover_password'),
+    path('profile/update/', UpdateProfileView.as_view(), name='update_profile'),
```

Comment:
My suggestion is to not use /update here. You can utilize ProfileView instead (change UpdateProfileView to ProfileView) to utilize the APIView from drf.